### ISSUE TITLE
What GPUs were used for training on ICDAR2017 HTR?

### ISSUE THREAD

**Issue by IuliiaSaveleva** [IntuitionMachines/OrigamiNet#4]
How many GPUs were used during training on ICDAR2017 HTR and what was their memory capacity or other important stats (if it is necessary to mention)?

**Comment by ASDen** [comment]
For the largest model and best results we used 8x 16GB V100s, but it is important to mention:
- You can already do very well with smaller models (see the readme)
- You can also do very well using a big model but small input resolution
- The code supports gradient checkpointing and gradient accumulation to simplify using smaller number of gpus with less amount of available memory
